presto-druid/src/main/java/com/facebook/presto/druid/DruidQueryGenerator.java

```diff
@@ -236,7 +236,12 @@ public DruidQueryGeneratorContext visitProject(ProjectNode node, DruidQueryGener
                         variable,
                         new Selection(druidExpression.getDefinition(), druidExpression.getOrigin()));
             });
-            return context.withProject(newSelections);
+
+            Map<VariableReferenceExpression, Selection> allSelections = context.getSelections().entrySet().stream()
```

Comment:
am not convinced by this change. for queries like:
select count(1) from table group by cityName

user need to specify cityName explicitly:
select cityName, count(1) from table group by cityName

We might not need to add cityName for user.